services/director-v2/src/simcore_service_director_v2/core/dynamic_services_settings/scheduler.py

```diff
@@ -30,7 +30,7 @@ class DynamicServicesSchedulerSettings(BaseCustomSettings):
     )
 
     DYNAMIC_SIDECAR_DOCKER_COMPOSE_VERSION: str = Field(
-        "3.8", description="docker-compose spec version used in the compose-specs"
+        "3.8", description="docker-compose spec version used in the compose-specs", deprecated=True
     )
```

Comment:
Q: What happens if I specify 2.4 here or nothing? How will it behave?

Reply:
I don't know. but since we are using the newer docker compose plugin. I think you should have a look. what I did here does not change anything.

Reply:
@GitHK I guess you should read this: https://docs.docker.com/compose/compose-file/04-version-and-name/